lib/conditionals.md

```diff
@@ -5,8 +5,19 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a conditional?
 
-Replace me with your response...
+A conditional statement is one that allows the computer to run the same code until it runs into a circumstance that has been predefined 
+afterwhich the command differs.  If I were programming a robot this would be a command something like:
+Move forward. If unable to move forward, then turn left. Move forward.
+
 
 # What is the difference between `=` and `==`?
 
-Replace me with your response...
\ No newline at end of file
+To me, these symbols seem to be linked similarly to similies and metaphors
+
+= is for telling a variable to point at an object, kind of like a metaphor. 
+She's a brick house.
+She = brick house
+
+== still maintains that the two inputs are seperate but equal, such as a similie. 
+We're glowing like the metal on the edge of a knife.
+We're glowing == the metal on the edge of a knife
```

Comment:
There is an important difference between simile and `==`. In simile, a lion and bravery likened to each other; but they are not _equal_ in the strictest sense. That is, we might say that someone is brave like a lion; but we would not say say that "bravery" and "lion" _are the same thing_.

- [ ] In Ruby, if we ran the code `"lion" == "bravery"`, what would the result be?

Other than this clarification, I'm comfortable with your abstract approach to understanding these concepts. But it would probably be wise to note down somewhere (even if only in your private notes) that `=` is often called the "assignment operator", and `==` is often called the "comparison operator".